ccloud/go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/confluentinc/ccloud-sdk-go-v1-public
+module github.com/confluentinc/ccloud-sdk-go-v1-public/ccloud
```

Comment:
Are you planning on putting another sub-package in this repo? What's the reason for this?

Reply:
Feels more organized this way as I was planning on adding v1 mds sdk. The plan is to only release v1 on-prem mds and put cloud role api in v1-public.

Reply:
It still feels weird cramming multiple repos in here, how do you feel about creating a mds-sdk-go-public for the MDS cloud SDK?

Reply:
ok let's do that!